webpack/assets/javascripts/react_app/components/DiffView/DiffContainer.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ButtonGroup, Button } from 'patternfly-react';
+
+import DiffView from './DiffView';
+import './diffview.scss';
+
+class DiffContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      viewType: 'split',
```

Comment:
the default is defined twice, here and in the DiffView component, should we define it only once (perhaps here?)